en/docs/learn/api-microgateway/deploy-apis-in-privatejet-mode.md

```diff
@@ -0,0 +1,321 @@
+# PrivateJet mode for Microgateways
+With many of applications gearing towards microservice architecture, it’s no surprise that container-orchestration systems such as Kubernetes have become so popular with functionalities such as automating computer application deployment, scaling, and management, 
+which simplifies a number of complex management tasks. WSO2 API-Manager provides cloud-native API management where user can expose microservices as managed APIs in cloud environments such as Kubernetes with the support of [WSO2-Kubernetes API Operator]({{base_path}}/learn/kubernetes-operators/k8s-api-operator.md). 
+
+ ![Architecture]({{base_path}}/assets/img/learn/privatejet-mode/architecture.png)
+ 
+Microservices will expose as managed APIs in cloud clusters in privetJet mode, where each microservice will have a dedicated [WSO2 API Microgateway](https://wso2.com/api-management/api-microgateway/). This will provide maximum security and guaranteed resource allocation for API execution. As depicted in the above diagram, When the APIs published via API-Manager in cloud environments,
+deployment, scaling, and management tasks will handle by the [WSO2 -Kubernetes API Operator]({{base_path}}/learn/kubernetes-operators/k8s-api-operator.md) itself.
+
+## Configuring APIs to deploy in PrivateJet mode
```

Comment:
Shouldn't this be, "Enabling PrivateJet mode"

Reply:
Changed to Enabling PrivateJet mode